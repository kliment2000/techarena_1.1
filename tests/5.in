3
12 5 1
2
1 a 3
2 b 1
2
1 a
2 b
0
