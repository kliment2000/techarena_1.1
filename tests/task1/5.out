(2b (1a 3)) 37.9
